apps/neuvector/neuvector/aat/00.yaml

```diff
@@ -76,7 +76,19 @@ spec:
           host: cft-neuvector00.aat.platform.hmcts.net
           enabled: true
           annotations:
+            kubernetes.io/ingress.class: traefik
             traefik.ingress.kubernetes.io/router.tls: "true"
+            traefik.ingress.kubernetes.io/router.middlewares: neuvector-neuvectorredirect@kubernetescrd
             traefik.ingress.kubernetes.io/frontend-entry-points: http,https
             traefik.ingress.kubernetes.io/redirect-entry-point: https
-            kubernetes.io/ingress.class: traefik
\ No newline at end of file
+---
+apiVersion: traefik.containo.us/v1alpha1
```

Comment:
You will need to move into a different file and add here https://github.com/hmcts/cnp-flux-config/blob/dxw/neuvector-https/apps/neuvector/aat/00/kustomization.yaml#L5

Reply:
yep, just changed this then saw your comment. thanks